ReportSimulationOutput/measure.rb

```diff
@@ -1318,6 +1318,13 @@ def report_runperiod_output_results(runner, outputs, output_format, annual_outpu
       @component_loads.each do |_load_type, load|
         results_out << ["#{load.name} (#{load.annual_units})", load.annual_output.to_f.round(n_digits)]
       end
+
+      sum_comp_htg = results_out.filter_map { |row| row[1] if (not row[0].nil?) && (row[0].start_with? 'Component Load: Heating:') }.sum()
+      sum_comp_clg = results_out.filter_map { |row| row[1] if (not row[0].nil?) && (row[0].start_with? 'Component Load: Cooling:') }.sum()
+      residual_htg = @loads[LT::Heating].annual_output.to_f - sum_comp_htg
+      residual_clg = @loads[LT::Cooling].annual_output.to_f - sum_comp_clg
+      results_out << ['Component Load: Heating: Residual (MBtu)', residual_htg.round(n_digits)]
+      results_out << ['Component Load: Cooling: Residual (MBtu)', residual_clg.round(n_digits)]
```

Comment:
This produces the Residual load for annual output. I assume similar code is needed for timeseries output?

Reply:
I debated whether to include a timeseries output for the residual. I am not sure it really makes sense at a timestep-level because the heating/cooling delivered does not change concurrently with the component loads. So we might see several hours of high error when the HVAC energy = 0, then a quick change in the opposite direction once the HVAC system turns on, roughly offsetting to 0 over the long term (hopefully).

The downside is that since it is reported as just another component load here, it may be confusing to not see in the timeseries.

Reply:
I see. Given that, I do wonder if it's really that useful to provide the Residual output. If anyone wants it, it can be easily calculated from the annual outputs that are provided.

Reply:
If there is any confusion as to the relationship between the two, I'm tempted to say we should improve the documentation rather than add another output, which would require even _more_ explanation.